Makefile

```diff
@@ -175,6 +175,10 @@ addlicense: $(ADDLICENSE) ## Download addlicense locally if necessary.
 $(ADDLICENSE): $(LOCALBIN)
 	test -s $(LOCALBIN)/addlicense || GOBIN=$(LOCALBIN) go install github.com/google/addlicense@$(ADDLICENSE_VERSION)
 
+.PHONY: integration-tests
```

Comment:
move this target near standard unit test target. This section primary for installation additional tools

Reply:
pls add some comment for use make help

Reply:
Done.